routers/repo/issue.go

```diff
@@ -1447,6 +1448,12 @@ func ChangeIssueReaction(ctx *context.Context, form auth.ReactionForm) {
 
 	switch ctx.Params(":action") {
 	case "react":
+		if !util.IsStringInSlice(form.Content, setting.UI.Reactions) {
+			log.Error("ChangeIssueReaction: " + form.Content + " is no allowed reaction")
```

Comment:
Logging this to the server log is not appropriate. The user needs to get this message.

Btw log messages must use %s for embedded variables. You get nice colour encoding and, it prevents % from being interpreted within the messages.

Reply:
nice info (%s) and fix it if i now how to show up an error message in UX -> https://github.com/go-gitea/gitea/pull/8886#discussion_r344468236